### PR TITLE
Edge support

### PR DESCRIPTION
**Pull request by japborst**
As Edge and Chrome share the same API, porting was relatively easy. `chrome` becomes `browser` and `storage.sync` can be replaced with `storage.local` as getters and setters work identical.

Included some additional improvements by moving the storage logic to one place.

Tested in both Edge and Chrome, both latest version. Edge requires an additional entry in the manifest, which Chrome warns about but does not break. Might require separate manifest files, but loading in Chrome went just fine. Not sure if that is the case when publishing in the Chrome web store?

Any feedback is appreciated.

### CODE REVIEW

**Comment by jianweichuah** [comment]
Hey Jelmer, thanks for the pull request and sorry for the delayed response as I have been busy at work recently. I'd like to do some testing on both Chrome and Edge first before merging. I'll provide an update once it's done. Thanks for contributing!

**Comment by japborst** [comment]
No worries. Thanks for the update. 

More than logical that you want to test it first. Let me know if you find some undesired behaviour.

File: popup.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ document.addEventListener("DOMContentLoaded", function(event) {
     onOffSwitch.onclick = toggleSwitch;
 
     function getActivationStatus(callBack) {
-        chrome.runtime.sendMessage({"get_activation_status": true}, function(response) {
+        browser.runtime.sendMessage({"get_activation_status": true}, function(response) {
```
Comment (jianweichuah) [inline popup.js:19]: Need to add:

```
// Try if Edge browser object exists, else default to chrome
var browser = self.browser;
if (typeof browser === "undefined") {
    browser = self.chrome;
}
```

above for the pop up to work.

**Review by jianweichuah — CHANGES_REQUESTED** [review]
There were warnings when trying to install this extension:
Unrecognized manifest key 'minimum_edge_version'.

I'm afraid the warning might be confusing to the users. Should we have a separate manifest for Chrome and Edge?

File: popup.js (hunk 1 above)
Comment (japborst) [reply popup.js:19]: Overlooked this one. Thanks!

**Comment by japborst** [comment]
Separated the manifests to be sure. So no warnings should be expected. Included build script for both chrome and tidied it a little bit in the process.

**Review by jianweichuah — APPROVED** [review]
Looks good!

**Pull request merged**